components/esp-tls/Kconfig

```diff
@@ -115,4 +115,12 @@ menu "ESP-TLS"
         help
             Enable detailed debug prints for wolfSSL SSL library.
 
+    config ESP_WOLFSSL_OCSP_CHECKALL
```

Comment:
Can you please move this config option to `esp-wolfssl` component?

Reply:
Yes, I was first confused with the name being ESP_WOLFSSL.
Maybe it would be better to change this config option as  `ESP_TLS_OCSP_CHECKALL`
Other things can remain the same